code/__DEFINES/colours.dm

```diff
@@ -1,214 +1,228 @@
-#define COLOUR_SOAPSTONE_PLASTIC "#a19d94"
-#define COLOUR_SOAPSTONE_IRON "#b2b2b2"
-#define COLOUR_SOAPSTONE_BRONZE "#FE8001"
-#define COLOUR_SOAPSTONE_SILVER "#FFFFFF"
-#define COLOUR_SOAPSTONE_GOLD "#FFD900"
-#define COLOUR_SOAPSTONE_DIAMOND "#00ffee"
-
-#define COLOUR_GREEN_GRAY    "#99BB76"
-#define COLOUR_RED_GRAY  "#B4696A"
-#define COLOUR_PALE_BLUE_GRAY   "#98C5DF"
-#define COLOUR_PALE_GREEN_GRAY  "#B7D993"
-#define COLOUR_PALE_RED_GRAY "#D59998"
-#define COLOUR_PALE_PURPLE_GRAY "#CBB1CA"
-#define COLOUR_PURPLE_GRAY   "#AE8CA8"
-
-//Colour defines used by the assembly detailer.
-#define COLOUR_ASSEMBLY_BLACK   "#545454"
-#define COLOUR_ASSEMBLY_BGRAY   "#9497AB"
-#define COLOUR_ASSEMBLY_WHITE   "#E2E2E2"
-#define COLOUR_ASSEMBLY_RED  "#CC4242"
-#define COLOUR_ASSEMBLY_ORANGE  "#E39751"
-#define COLOUR_ASSEMBLY_BEIGE   "#AF9366"
-#define COLOUR_ASSEMBLY_BROWN   "#97670E"
-#define COLOUR_ASSEMBLY_GOLD "#AA9100"
-#define COLOUR_ASSEMBLY_YELLOW  "#CECA2B"
-#define COLOUR_ASSEMBLY_GURKHA  "#999875"
-#define COLOUR_ASSEMBLY_LGREEN  "#789876"
-#define COLOUR_ASSEMBLY_GREEN   "#44843C"
-#define COLOUR_ASSEMBLY_LBLUE   "#5D99BE"
-#define COLOUR_ASSEMBLY_BLUE "#38559E"
-#define COLOUR_ASSEMBLY_PURPLE  "#6F6192"
-
-///Colours for xenobiology vatgrowing
-#define COLOUR_SAMPLE_YELLOW "#c0b823"
-#define COLOUR_SAMPLE_PURPLE "#342941"
-#define COLOUR_SAMPLE_GREEN "#98b944"
-#define COLOUR_SAMPLE_BROWN "#91542d"
-#define COLOUR_SAMPLE_GRAY "#5e5856"
-
-///Main colours for UI themes
-#define COLOUR_THEME_MIDNIGHT "#6086A0"
-#define COLOUR_THEME_PLASMAFIRE "#FFB200"
-#define COLOUR_THEME_RETRO "#24CA00"
-#define COLOUR_THEME_SLIMECORE "#4FB259"
-#define COLOUR_THEME_OPERATIVE "#B8221F"
-#define COLOUR_THEME_GLASS "#75A4C4"
-#define COLOUR_THEME_CLOCKWORK "#CFBA47"
-
-///Colours for eigenstates
-#define COLOUR_PERIWINKLEE "#9999FF"
+#define CL_COLOR_PRIORITY_AMOUNT 4
+
+#define CL_COLOR_DARKMODE_BACKGROUND "#202020"
+#define CL_COLOR_DARKMODE_DARKBACKGROUND "#171717"
+#define CL_COLOR_DARKMODE_TEXT "#a4bad6"
+
+#define CL_COLOR_WHITE "#FFFFFF"
+#define CL_COLOR_VERY_LIGHT_GRAY "#EEEEEE"
+#define CL_COLOR_SILVER "#C0C0C0"
+#define CL_COLOR_GRAY "#808080"
+#define CL_COLOR_FLOORTILE_GRAY "#8D8B8B"
+#define CL_COLOR_DARK "#454545"
+#define CL_COLOR_ALMOST_BLACK "#333333"
+#define CL_COLOR_BLACK "#000000"
+#define CL_COLOR_HALF_TRANSPARENT_BLACK "#0000007A"
+
+#define CL_COLOR_RED "#FF0000"
+#define CL_COLOR_MOSTLY_PURE_RED "#FF3300"
+#define CL_COLOR_DARK_RED "#A50824"
+#define CL_COLOR_RED_LIGHT "#FF3333"
+#define CL_COLOR_MAROON "#800000"
+#define CL_COLOR_VIVID_RED "#FF3232"
+#define CL_COLOR_LIGHT_GRAYISH_RED "#E4C7C5"
+#define CL_COLOR_SOFT_RED "#FA8282"
+#define CL_COLOR_CULT_RED "#960000"
+#define CL_COLOR_BUBBLEGUM_RED "#950A0A"
+
+#define CL_COLOR_YELLOW "#FFFF00"
+#define CL_COLOR_VIVID_YELLOW "#FBFF23"
+#define CL_COLOR_VERY_SOFT_YELLOW "#FAE48E"
+
+#define CL_COLOR_OLIVE "#808000"
+#define CL_COLOR_VIBRANT_LIME "#00FF00"
```

Comment:
VIBRANT_LIME, duplicate of existing COLOR_GREEN

Reply:
isn't used and is duplicated deleted.